source/_integrations/motionblinds_ble.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: Motionblinds BLE
+description: Instructions on how to integrate Motionblinds Bluetooth motors into Home Assistant.
+ha_category:
+  - Cover
+ha_iot_class: Assumed State
+ha_release: 2024.04
+ha_domain: motionblinds_ble
+ha_codeowners:
+  - '@LennP'
+  - '@jerrybboy'
+ha_config_flow: true
+ha_platforms:
+  - cover
+ha_integration_type: integration
+---
+
+This integration adds support for [Motionblinds](https://motionblinds.com/) Bluetooth motors. Beware that this integration does not work with *Eve Motionblinds* motors. *Eve Motionblinds* can be added to Home Assistant using the [HomeKit Device](https://www.home-assistant.io/integrations/homekit_controller/) integration.
```

Comment:
```suggestion
The Motionblinds BLE {% term integration %} adds support for [Motionblinds](https://motionblinds.com/) Bluetooth motors. Beware that this integration does not work with *Eve Motionblinds* motors. *Eve Motionblinds* can be added to Home Assistant using the [HomeKit Device](https://www.home-assistant.io/integrations/homekit_controller/) integration.
```

Reply:
`Beware that this integration does not work with *Eve Motionblinds* motors. `
Does this refer to non-Bluetooth motors?

Reply:
Eve Motionblinds do use bluetooth but they are Homekit devices and do not work with this integration

Reply:
Thank you for clarifying.